### ISSUE TITLE
I'm unable to change the file name

### ISSUE THREAD

**Issue by zeeshansaiyed** [jgilfelt/android-simpl3r#3]
how to change the name of file I'm uploading ?, it's by default to value of s3key

**Comment by achinverma** [comment]
yes i am having the same issue , i also want to change the file name but didn't get any hint

**Comment by ryanbennettvoid** [comment]
s3key is the file name.

Not to be confused with the access key which is for authorization.

**Comment by achinverma** [comment]
hi, thanks for reply, but when i try to upload a file xyz.pdf  it is successfully uploaded to my bucket but when i look into my bucket it is saved in XYZ.bin format , after uploading any file it's format changed to .bin please tell me where i am wrong?

**Comment by ryanbennettvoid** [comment]
If you're using Android Studio, press CTRL+SHIFT+F, then type **.bin** to search your whole project.

**Comment by achinverma** [comment]
yes i have search in my whole project but did not find .bin in my project , is it possible to set content type of the file via code when uploading any file to server?
